navigation/navigation-common/src/androidTest/java/androidx/navigation/NavDeepLinkTest.kt

```diff
@@ -320,14 +317,14 @@ class NavDeepLinkTest {
 
         val matchArgs = deepLink.getMatchingArguments(
             Uri.parse("$DEEP_LINK_EXACT_HTTPS/users"),
-            mapOf("myarg" to nullableStringArgument())
+            mapOf("myarg" to nullableStringArgument(null))
         )
         assertWithMessage("Args should not be null")
             .that(matchArgs)
             .isNotNull()
-        assertWithMessage("Args should contain the argument and it should be null")
-            .that(matchArgs?.getString("myarg"))
-            .isNull()
+        assertWithMessage("Args should not contain the argument")
+            .that(matchArgs?.containsKey("myarg"))
+            .isFalse()
```

Comment:
There were many tests like this that actually did not what was declared in assert

Reply:
Hi, could you please put all the test updating into a separate pull request.

Reply:
Moved it to #210